osu.Game/Overlays/OnlineOverlay.cs

```diff
@@ -32,7 +32,7 @@ protected OnlineOverlay(OverlayColourScheme colourScheme)
                         Direction = FillDirection.Vertical,
                         Children = new Drawable[]
                         {
-                            Header,
+                            Header.With(h => h.Depth = -float.MaxValue),
```

Comment:
That negative max value really feels like what caused the confusion. 
```suggestion
                            Header.With(h => h.Depth = float.MinValue),
```